package.json

```diff
@@ -14,10 +14,11 @@
     "axios": "^1.4.0",
     "expo": "~48.0.15",
     "expo-checkbox": "~2.3.1",
+    "expo-secure-store": "~12.1.1",
     "expo-status-bar": "~1.4.4",
     "react": "18.2.0",
     "react-native": "0.71.7",
-    "react-native-gesture-handler": "^2.10.0",
```

Comment:
interesting, why did you need to downgrade?

Reply:
It was something expo suggested me, it said some things would not work properly if I use that version and suggested me which one to install instead